examples/upperair/IGRA2_Request.py

```diff
@@ -0,0 +1,59 @@
+# Copyright (c) 2018 University Corporation for Atmospheric Research/Unidata.
+# Distributed under the terms of the MIT License.
+# SPDX-License-Identifier: MIT
+"""
+IGRA2 Upper Air Data Request
+==============================
+
+This example shows how to use siphon's `simplewebswervice` support to create a query to
+the Integrated Global Radiosonde Archive version 2.
+"""
+
+from datetime import datetime
+
+from siphon.simplewebservice.igra2 import IGRAUpperAir
+
+####################################################
+# Create a datetime object for the sounding and string of the station identifier.
+date = datetime(2014, 9, 10, 0)
+station = 'USM00070026'
+
+####################################################
+# Make the request. IGRAUpperAir returns a dataframe containing the sounding data and
+# a dataframe with station metadata from the sounding header.
+df, header = IGRAUpperAir.request_data(date, station)
+
+####################################################
+# Inspect data columns in the dataframe.
+print(df.columns)
```

Comment:
These `print`s should work just fine as is on Python 2, since the parenthesis that mean function call on Python 3 just mean "group this" on Python 2.